src/error.rs

```diff
@@ -92,6 +92,7 @@ impl std::ops::Deref for Error {
 
 /// The types of errors that can occur.
 #[derive(Debug, Error)]
+#[non_exhaustive]
 pub enum ErrorKind {
```

Comment:
ditto considering adding `non_exhaustive` to any enum variants here.

Reply:
Done